src/WebJobs.Script/Workers/Rpc/FunctionRegistration/RpcFunctionInvocationDispatcher.cs

```diff
@@ -143,24 +154,47 @@ internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount
             }
         }
 
+        internal async Task InitializeJobhostLanguageWorkerChannelAsync(int attemptCount, IEnumerable<string> languages)
+        {
+            foreach (string language in languages)
+            {
+                var rpcWorkerChannel = _rpcWorkerChannelFactory.Create(_scriptOptions.RootScriptPath, language, _metricsLogger, attemptCount, _workerConfigs);
+                _jobHostLanguageWorkerChannelManager.AddChannel(rpcWorkerChannel, language);
+                await rpcWorkerChannel.StartWorkerProcessAsync();
+                _logger.LogDebug("Adding jobhost language worker channel for runtime: {language}. workerId:{id}", language, rpcWorkerChannel.Id);
```

Comment:
How does this work? a format string usually has numerical placeholders. An interpolated string would have the $ start marker.

Reply:
It's logged this way in whole function's code. They are using [`MSLogger`](https://docs.microsoft.com/en-us/dotnet/core/extensions/logging?tabs=command-line#create-logs)